test/e2e/cert-manager_test.go

```diff
@@ -0,0 +1,72 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	certmanager "go.etcd.io/etcd-operator/pkg/certificate/cert-manager"
+	interfaces "go.etcd.io/etcd-operator/pkg/certificate/interfaces"
+)
+
+func TestCertManagerCM(t *testing.T) {
+	feature := features.New("Cert-Manager Certificate").WithLabel("app", "cert-manager")
+
+	feature.Setup(
+		func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			cmProvider := certmanager.New()
+			_, getCertConfigErr := cmProvider.GetCertificateConfig(ctx, "test-cert-secret", cfg.Namespace())
```

Comment:
You create a `cmProvider`, then call `GetCertificateConfig`, but you do not provide any parameter, how it can work?